### PR TITLE
Adding additional PN versions

### PR DESCRIPTION
**Pull request by wpmccormack**
#### PR description:

Adding other PN versions: MD-2prong, mass regression, and AK4.

### CODE REVIEW

File: RecoBTag/ONNXRuntime/python/pfParticleNetAK4_cff.py [hunk 1]
```diff
@@ -18,6 +19,22 @@
...
 )
 
+pfParticleNetAK4SonicJetTags = particleNetSonicJetTagsProducer.clone(
+    src = 'pfParticleNetAK4TagInfos',
+    preprocess_json = 'RecoBTag/Combined/data/ParticleNetAK4/CHS/V00/preprocess_noragged.json',
+    Client = cms.PSet(
+        timeout = cms.untracked.uint32(300),
+        modelName = cms.string("particlenet_AK4"),
+        mode = cms.string("Async"),
+        #mode = cms.string("Sync"),
```
Comment (kpedro88) [inline RecoBTag/ONNXRuntime/python/pfParticleNetAK4_cff.py:29]: delete

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py [hunk 2]
```diff
@@ -28,9 +28,10 @@
         timeout = cms.untracked.uint32(300),
         modelName = cms.string("particlenet"),
         mode = cms.string("Async"),
+        #mode = cms.string("Sync"),
```
Comment (kpedro88) [inline RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py:31]: delete

File: RecoBTag/ONNXRuntime/python/pfParticleNetAK4_cff.py [hunk 3]
```diff
@@ -18,6 +19,22 @@
...
+        modelConfigPath = cms.FileInPath("HeterogeneousCore/SonicTriton/data/models/particlenet_AK4/config.pbtxt"),
+        modelVersion = cms.string(""),
+        verbose = cms.untracked.bool(True),
```
Comment (kpedro88) [inline RecoBTag/ONNXRuntime/python/pfParticleNetAK4_cff.py:32]: set to False for production

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py [hunk 4]
```diff
@@ -28,9 +28,10 @@
...
         modelConfigPath = cms.FileInPath("HeterogeneousCore/SonicTriton/data/models/particlenet/config.pbtxt"),
         modelVersion = cms.string(""),
-        verbose = cms.untracked.bool(False),
+        verbose = cms.untracked.bool(True),
```
Comment (kpedro88) [inline RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py:34]: set to False for production

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py [hunk 5]
```diff
@@ -46,13 +47,46 @@
...
 )
 
+pfMassDecorrelatedParticleNetSonicJetTags = particleNetSonicJetTagsProducer.clone(
+    src = 'pfParticleNetTagInfos',
+    preprocess_json = 'RecoBTag/Combined/data/ParticleNetAK8/MD-2prong/V01/preprocess_noragged.json',
+    Client = cms.PSet(
+        timeout = cms.untracked.uint32(300),
+        modelName = cms.string("particlenet_AK8_MD-2prong"),
+        mode = cms.string("Async"),
+        #mode = cms.string("Sync"),
```
Comment (kpedro88) [inline RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py:57]: delete

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py [hunk 6]
```diff
@@ -46,13 +47,46 @@
...
+        modelConfigPath = cms.FileInPath("HeterogeneousCore/SonicTriton/data/models/particlenet_AK8_MD-2prong/config.pbtxt"),
+        modelVersion = cms.string(""),
+        verbose = cms.untracked.bool(True),
```
Comment (kpedro88) [inline RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py:60]: set to False for production

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py [hunk 7]
```diff
@@ -46,13 +47,46 @@
...
 )
 
+pfParticleNetMassRegressionSonicJetTags = particleNetSonicJetTagsProducer.clone(
+    src = 'pfParticleNetTagInfos',
+    preprocess_json = 'RecoBTag/Combined/data/ParticleNetAK8/MassRegression/V01/preprocess_noragged.json',
+    Client = cms.PSet(
+        timeout = cms.untracked.uint32(300),
+        modelName = cms.string("particlenet_AK8_MassRegression"),
+        mode = cms.string("Async"),
+        #mode = cms.string("Sync"),
```
Comment (kpedro88) [inline RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py:81]: delete

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py [hunk 8]
```diff
@@ -46,13 +47,46 @@
...
+        modelConfigPath = cms.FileInPath("HeterogeneousCore/SonicTriton/data/models/particlenet_AK8_MassRegression/config.pbtxt"),
+        modelVersion = cms.string(""),
+        verbose = cms.untracked.bool(True),
```
Comment (kpedro88) [inline RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py:84]: set to False for production

File: RecoBTag/ONNXRuntime/python/pfParticleNetAK4_cff.py (hunk 1 above)
Comment (wpmccormack) [reply RecoBTag/ONNXRuntime/python/pfParticleNetAK4_cff.py]: Deleted

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py (hunk 2 above)
Comment (wpmccormack) [reply RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py]: Deleted

File: RecoBTag/ONNXRuntime/python/pfParticleNetAK4_cff.py (hunk 3 above)
Comment (wpmccormack) [reply RecoBTag/ONNXRuntime/python/pfParticleNetAK4_cff.py]: Done!

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py (hunk 4 above)
Comment (wpmccormack) [reply RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py]: Done!

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py (hunk 5 above)
Comment (wpmccormack) [reply RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py]: Deleted

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py (hunk 6 above)
Comment (wpmccormack) [reply RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py]: Done!

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py (hunk 7 above)
Comment (wpmccormack) [reply RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py]: Deleted

File: RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py (hunk 8 above)
Comment (wpmccormack) [reply RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py]: Done!

**Pull request merged**